Penteract/Assets/Scripts/GameController.h

```diff
@@ -23,6 +23,12 @@ class GameController : public Script {
 	void Rotate(float2 mouseMotion, Frustum* frustum, ComponentTransform* transform);
 	void PauseGame();
 	void ResumeGame();
+
+	static bool IsGameplayBlocked();					// Getter for isGameplayBlocked
```

Comment:
I would make these getters static bool const so you make sure no one will make that as a reference and modify the variable. But that's just missusage of the Getter, so let's hope that doesn't happend. Don't change it